diff --git a/dist/extensions-Co4y7P63.js b/dist/extensions-Co4y7P63.js
index 3ab79ac5b087d2034b9014dcb853252657074916..eab970066864a1bcac04008331eca2c1ac7c6086 100644
--- a/dist/extensions-Co4y7P63.js
+++ b/dist/extensions-Co4y7P63.js
@@ -1265,7 +1265,9 @@ var J = class {
 		if (t.length === 0) return null;
 		let n = t[0], r = Number.MAX_VALUE;
 		return t.forEach((t) => {
-			let i = t.querySelector(".bn-block-group").getBoundingClientRect(), a = e.clientX < i.left ? i.left - e.clientX : e.clientX > i.right ? e.clientX - i.right : 0, o = e.clientY < i.top ? i.top - e.clientY : e.clientY > i.bottom ? e.clientY - i.bottom : 0, s = Math.sqrt(a ** 2 + o ** 2);
+			let bg = t.querySelector(".bn-block-group");
+			if (!bg) return;
+			let i = bg.getBoundingClientRect(), a = e.clientX < i.left ? i.left - e.clientX : e.clientX > i.right ? e.clientX - i.right : 0, o = e.clientY < i.top ? i.top - e.clientY : e.clientY > i.bottom ? e.clientY - i.bottom : 0, s = Math.sqrt(a ** 2 + o ** 2);
 			s < r && (r = s, n = t);
 		}), {
 			element: n,
